src/test/kotlin/com/zepben/evolve/streaming/mutations/SwitchStateClientTest.kt

```diff
@@ -9,57 +9,120 @@
 package com.zepben.evolve.streaming.mutations
 
 import com.zepben.evolve.streaming.grpc.CaptureLastRpcErrorHandler
+import com.zepben.evolve.streaming.grpc.GrpcChannel
 import com.zepben.protobuf.nm.SetCurrentSwitchStatesRequest
 import com.zepben.protobuf.nm.SwitchStateServiceGrpc
 import com.zepben.testutils.junit.SystemLogExtension
+import io.grpc.Channel
 import io.grpc.Status
 import io.grpc.StatusRuntimeException
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.mockkStatic
+import io.mockk.verify
 import org.hamcrest.MatcherAssert.assertThat
 import org.hamcrest.Matchers.equalTo
 import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.extension.RegisterExtension
-import org.mockito.Mockito
-import org.mockito.Mockito.mock
-import org.mockito.Mockito.verify
 
 internal class SwitchStateClientTest {
     @JvmField
     @RegisterExtension
     var systemOut: SystemLogExtension = SystemLogExtension.SYSTEM_OUT.captureLog().muteOnSuccess()
 
-    private val stub = mock(SwitchStateServiceGrpc.SwitchStateServiceBlockingStub::class.java)
+    private val stub: SwitchStateServiceGrpc.SwitchStateServiceBlockingStub = mockk(relaxed = true)
```

Comment:
Changed as such, and also removed the type annotation for `client` since it's also private and it's pretty obvious that it's a `SwitchStateClient` from its definition.